packages/gatsby-cli/src/reporter/prepare-stack-trace.ts

```diff
@@ -5,14 +5,16 @@
 import { readFileSync, readdirSync } from "fs"
 import { codeFrameColumns } from "@babel/code-frame"
 import stackTrace from "stack-trace"
-import {
-  SourceMapConsumer,
-  BasicSourceMapConsumer,
-  IndexedSourceMapConsumer,
-  NullableMappedPosition,
-} from "source-map"
+import { TraceMap, originalPositionFor } from "@jridgewell/trace-mapping"
 import * as path from "path"
 
+interface INullableMappedPosition {
```

Comment:
Nit: We provide `OriginalMapping` and `InvalidOriginalMapping` types that can be used for discriminated unions.

Reply:
Good to know 👍